modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -721,6 +725,11 @@ NSFVBase<BaseType>::validParams()
       "'porous_medium_treatment' is enabled, the coefficients already contain a velocity "
       "multiplier but they are not multiplied with density yet!");
 
+  params.addParam<bool>(
+      "use_standard", true, "Flag to enable standard friction coefficient formulation");
```

Comment:
For the action we need more descriptive names. To make sure everybody knows this is for the friction parts.

Reply:
renamed the flag to make it more clear.